modules/ROOT/pages/kernel-args.adoc

```diff
@@ -109,6 +109,10 @@ systemd:
         # We run after `systemd-machine-id-commit.service` to ensure that
         # `ConditionFirstBoot=true` services won't rerun on the next boot.
         After=systemd-machine-id-commit.service
+        # Since this unit creates an rpm-ostree transaction, we run before
+        # `boot-complete.service` to ensure that there are no other conflicting
+        # transactions that might cause this unit to fail.
+        Before=boot-complete.service
```

Comment:
Note that `systemd-machine-id-commit.service` is ordered `After=` the similarly named `first-boot-complete.service`, and so this would create a cycle if we ordered `Before=first-boot-complete.service`.

But I think `boot-complete.service` is independent of `first-boot-complete.service` so we should be OK here.